content/dstu2.md

```diff
@@ -37,7 +37,202 @@ Status: 200 OK
+{
+  "resourceType": "Conformance",
+  "text": {
+    "status": "generated",
+    "div": "\u003Cdiv\u003EGenerated Conformance Statement\u003C/div\u003E"
```

Comment:
Ah, yeah I didn't realize that the \u003C 's weren't getting replaced. Fixed:
https://github.com/cerner/fhir.cerner.com/pull/70/commits/6c675aca11f6732e37b7eded14506093bf59bb88